EIPS/eip-1.md

```diff
@@ -21,69 +21,66 @@ For Ethereum implementers, EIPs are a convenient way to track the progress of th
+```
+[ WIP ] -> [ DRAFT ] -> [ LAST CALL ] -> [ ACCEPTED ] -> [ FINAL ]
+```
 
-EIPs can also be superseded by a different EIP, rendering the original obsolete.
+Each status change is requested by the EIP author by @-mentioning or emailing the EIP editors. The EIP editors will process these requests as per the conditions below.
 
-The possible paths of the status of EIPs are as follows:
+* **Work in progress (WIP)** -- Once the champion has asked the Ethereum community whether an idea has any chance of acceptance, they will write a draft EIP as a [pull request]. Consider including an implementation if this will aid people in studying the EIP.
+  * :arrow_right: Draft -- If approved, EIP editor will assign the EIP a number (generally the issue or PR number related to the EIP) and merge your pull request. The EIP editor will not unreasonably deny an EIP.
+  * :x: Draft -- Reasons for denying draft status include being too unfocused, too broad, duplication of effort, being technically unsound, not providing proper motivation or addressing backwards compatibility, or not in keeping with the [Ethereum philosophy](https://github.com/ethereum/wiki/wiki/White-Paper#philosophy).
+* **Draft** -- Once the first draft has been merged, you may submit follow-up pull requests with further changes to your draft until such point as you believe the EIP to be mature and ready to proceed to the next status. An EIP in draft status must have implementations to be considered for promotion to the next status.
+  * :arrow_right: Last Call -- If approved, the EIP editor will assign Last Call status and set a review end date, normally 14 days later.
+  * :x: Last Call -- A request for Last Call status will be denied if material changes are still expected to be made to the draft. We hope that EIPs only enter Last Call once, so as to avoid unnecessary noise on the RSS feed. Last Call will be denied if the implementation is not complete and accepted by the community.
+* **Last Call** -- This EIP will listed prominently on the http://eips.ethereum.org/ website (subscribe via RSS at [last-call.xml](/last-call.xml)).
+  * :x: -- A Last Call which results in material changes or substantial unaddressed complaints will cause the EIP to revert to Draft or worse.
+  * :arrow_right: Accepted (Core EIPs only) -- After the review end date, the Ethereum Core Developers will vote on whether to accept this change. If yes, the status will upgrade to Accepted.
+  * :arrow_right: Final (Not core EIPs) -- A successful Last Call without material changes or unaddressed complaints will become Final.
+* **Accepted (Core EIPs only)** -- This is being implemented by Ethereum Core Developers.
+  * :arrow_right: Final -- When a Standards Track Core EIP is implemented in at least three viable Ethereum clients, and it is deployed on at least 50% of deployed nodes, and the implementations pass a common set of test suites, then the status is upgraded to Final.
```

Comment:
It's an open question whether this should be the case, or whether Final should wait for a hard fork. What about the case where there's a contentious fork that gets rejected?

Reply:
For my reference, notes [from @Souptacular on this topic](https://github.com/ethereum/EIPs/pull/956#issuecomment-392918317) are:

> only final after the hard fork block # is reached